src/lib/transformers.js

```diff
@@ -0,0 +1,40 @@
+const colors = require('colors');
+
+/* istanbul ignore next */
+const padd = (v) => ` ${v} `;
+
+/* istanbul ignore next */
+module.exports.defaultTransformers = {
+    cl_red: s => colors.red(s),
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/loggin-js/loggin-js/pullRequest?prid=7629800)
```suggestion
    cl_red: (s) => colors.red(s),
```